coml/deconstruct.py

```diff
@@ -0,0 +1,993 @@
+# deconstructe svg to spec
```

Comment:
This deconstruct.py is part of `vis_utils.py`.

You can create a folder called `vis_utils`, and put `deconstruct.py` into that folder, and the original `vis_utils.py` as `vis_utils/verifier.py`.

Reply:
I don't want to check the content of this file. I'm assuming it's self-contained.